test/test_deposit_switch.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'test_helper'
+
+class PlaidDepositSwitchTest < PlaidTest
```

Comment:
double-checking: are any of the deposit switch API response fields optional? if so we shouldn't `refute_empty` on them because it will create flaky tests.

Reply:
there are, but I have omitted the optional fields! :)